include/mbedtls/ccm.h

```diff
@@ -1,10 +1,19 @@
 /**
  * \file ccm.h
  *
- * \brief Counter with CBC-MAC (CCM) for 128-bit block ciphers
+ * \brief AES CCM combines Counter mode encryption with CBC-MAC authentication 
+ * for 128-bit block ciphers.
+ *  
+ * Input to CCM includes the following elements:
+ * <ul><li>Payload - text data that is both authenticated and encrypted.</li>
+ * <li>Associated data (Adata) - data that is authenticated but not 
+ * encrypted, For example, a header.</li>
+ * <li>Nonce - A unique value that is assigned to the payload and the 
+ * associated data.</li></ul>
+ * 
  */
-/*
- *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+
```

Comment:
I think this style change should be made either in all or in no files; currently the copyright comments have line breaks before the actual copyright statement.

Reply:
fixed

Reply:
ok